### PR TITLE
Better: Add created_at filtering for Attachment API RD-22307

### PR DESCRIPTION
**Pull request by kalilz4485**

### CODE REVIEW

File: specs/engage-digital_openapi3.yaml [hunk 1]
```diff
@@ -267,6 +267,20 @@ paths:
...
+          required: false
+          schema:
+            type: date
+        - description: >-
+            Search parameter to get attachments created after a ISO 8601 date
+          in: query
+          name: created_after
+          required: false
+          schema:
+            type: date
```
Comment (kalilz4485) [inline specs/engage-digital_openapi3.yaml:283]: Screenshot from 2022-07-28 17-16-04

Comment (kalilz4485) [reply specs/engage-digital_openapi3.yaml:283]: Screenshot from 2022-07-28 17-17-48

Comment (ylecuyer) [reply specs/engage-digital_openapi3.yaml:283]: Thanks

File: specs/engage-digital_openapi3.yaml [hunk 2]
```diff
@@ -267,6 +267,20 @@ paths:
           required: false
           schema:
             type: string
+        - description: >-
+            Search parameter to get attachments created before a ISO 8601 date
```
Comment (ylecuyer) [inline specs/engage-digital_openapi3.yaml:271]:
```suggestion
            Search parameter to get attachments created before a ISO 8601 datetime (inclusive)

```

File: specs/engage-digital_openapi3.yaml [hunk 3]
```diff
@@ -267,6 +267,20 @@ paths:
...
             type: string
+        - description: >-
+            Search parameter to get attachments created before a ISO 8601 date
+          in: query
+          name: created_before
+          required: false
+          schema:
+            type: date
+        - description: >-
+            Search parameter to get attachments created after a ISO 8601 date
```
Comment (ylecuyer) [inline specs/engage-digital_openapi3.yaml:278]:
```suggestion
            Search parameter to get attachments created after a ISO 8601 datetime (inclusive)

```

**Comment by ylecuyer** [comment]
LGTM

**Pull request merged**